src/create_disk.sh

```diff
@@ -467,6 +467,15 @@ chroot_run() {
     done
 }
 
+generate_gpgkeys() {
+    tmp_home=$(mktemp -d /tmp/gpg-XXXXXX)
+    gpg --homedir "${tmp_home}" --batch --passphrase '' --yes --quick-gen-key secex default
+    gpg --homedir "${tmp_home}" --armor --export secex > "/srv/builds/latest/s390x/ignition.gpg.pub"
```

Comment:
We shouldn't write to the final build dir from this script. Also the cosa workdir may not always be at `/srv`.

Here's a suggestion: add a new `--write-ignition-pubkey-to` switch to `create_disk.sh` which takes the path to which we write the pubkey. In `buildextend-metal`, pass that switch when building the secex image using a path under the temporary build dir, and then move the key into the final build dir when finalizing artifacts.

Reply:
The buildid might also not be latest. We might do a buildextend with `--build`